core/src/main/java/com/microsoft/applicationinsights/internal/config/ParamExcludedTypeXmlElement.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.applicationinsights.internal.config;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+//This is the class for binding the xml array list of <ExcludedTypes>
```

Comment:
It is better to use javadoc here, not just comment.

Reply:
@gavlyukovskiy yes we can use javadoc instead. I will update. Also we are planning to have a robust style guide to follow for this project in upcoming week and that should help in reducing this kind of inconsistency. Thanks for the suggestion,